### PR TITLE
updated domain lexicons and test for domain

### PR DESCRIPTION
**Pull request by carter-harris**
Updated Domain lexicons and wrote test for the domain

### CODE REVIEW

**Comment by keithbradley1** [comment]
great commit, these look like the type of tests we need 👍

File: test_domain_identifier.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  ''' Receives strings to be searched in the dictionary '''
+
+  @classmethod
+  def setUpClass(self):
+    self.domain = Domain_Identifier()
+    pass
+
+
+
+  # ----------------------------- WAR -------------------------------------------
```
Comment (Jessicashinjo) [inline test_domain_identifier.py]: Should be Healthcare

**Comment by Jessicashinjo** [comment]
I just saw one line that needed to be renamed and I put a comment by it.

**Comment by Jessicashinjo** [comment]
Everything looks good. We can wait to refactor until we have all of our test drafts completed.

**Pull request merged**